plugins/main/server/start/sanitize-uploaded-files/sanitize-svg.ts

```diff
@@ -0,0 +1,104 @@
+ * @param context
+ * @returns
+ */
+export default function sanitizeUploadedSVG(context) {
+  // Create a wrapper function that logs to plugin files and platform logging system
+  const createLog = (level: string) => message => {
+    log('sanitize:sanitizeUploadedSVG', message, level);
+    context.wazuh.logger[level](`sanitize:sanitizeUploadedSVG: ${message}`);
+  };
+
+  // Create the logger
+  const logger = {
+    info: createLog('info'),
+    warn: createLog('warn'),
+    error: createLog('error'),
+    debug: createLog('debug'),
+  };
+
+  try {
+    logger.info('Task sanitize SVG started');
+
+    logger.debug('Get Wazuh configuration');
+    const configuration = getConfiguration();
+    const logosSettingKeys = [
+      'customization.logo.sidebar',
+      'customization.logo.app',
+      'customization.logo.healthcheck',
+    ];
+
+    logger.debug('Check configuration for custom branding uploaded SVG files');
+    // Check each of the possible custom settings uploaded files look for SVG to sanitize
+    logosSettingKeys.forEach(logoKey => {
+      const logoSetting: TPluginSetting | undefined = PLUGIN_SETTINGS[logoKey];
+      const customLogoPath = configuration[logoKey];
+      if (!logoSetting || !customLogoPath) {
+        logger.debug(`Logo [${logoKey}] not customized. Skip.`);
+        return;
+      }
+
+      const targetDirectory = path.join(
+        __dirname,
+        '../../..',
+        logoSetting.options.file.store.relativePathFileSystem,
+      );
+
+      // If the setting folder doesn't exist abort
+      if (!fs.existsSync(targetDirectory)) {
+        return;
+      }
+
+      // Get the files related to the setting
+      const files = glob.sync(path.join(targetDirectory, `${logoKey}.*`));
+
+      // If there are no files saved abort
+      if (!files?.length) {
+        return;
+      }
+
+      // Check file extension
+      const fileName = path.basename(files[0]);
+      const fileExtension = path.extname(fileName);
+      if (fileExtension.toLocaleLowerCase() !== '.svg') {
+        return;
+      }
+
+      // Read the file contents
+      const fileFullPath = path.join(targetDirectory, fileName);
+      const originalFileBuffer = fs.readFileSync(fileFullPath);
+
+      // Sanitize the file contents
+      const originalSVGString = originalFileBuffer.toString();
+      const cleanSVG = sanitizeSVG(originalSVGString);
+      const cleanFileBuffer = Buffer.from(cleanSVG);
+
+      // Check if any changes were made in the sanitization process ignoring white spaces
+      // If any change was made then save the sanitized content
+      if (originalSVGString.replace(/\s/g, '') != cleanSVG.replace(/\s/g, '')) {
+        logger.info(
+          `[sanitize:sanitizeUploadedSVG] ${fileName} SVG file sanitized`,
+        );
+        // Delete the original file
+        fs.unlinkSync(fileFullPath);
+
+        // Save the clean file in the target directory
+        fs.writeFileSync(fileFullPath, cleanFileBuffer);
```

Comment:
**nitpick**: the logo customization settings when configured through the UI, the setting value includes a `v` with the date as the value that avoids when replacing the customization logo, the browser that accessed previously, downloads the new file instead of caching. So, for this use cases, we should update the `v` parameter of the logo setting value.